install/installer/pkg/components/server/configmap.go

```diff
@@ -253,9 +253,12 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 		ChargebeeProviderOptionsFile: fmt.Sprintf("%s/providerOptions", chargebeeMountPath),
 		StripeSecretsFile:            fmt.Sprintf("%s/apikeys", stripeSecretMountPath),
 		InsecureNoDomain:             false,
-		PrebuildLimiter: map[string]int{
+		PrebuildLimiter: PrebuildRateLimiters{
 			// default limit for all cloneURLs
-			"*": 50,
+			"*": PrebuildRateLimiterConfig{
+				Limit:  50,
+				Period: 300,
+			},
```

Comment:
Wanna extend the `configmap_test.go` to check for this? Given there was no change to the test file with this change suggests there's no test coverage for this.